### DISCUSSION TITLE
v6.0.0

### DISCUSSION THREAD

**Discussion by black7375** [black7375/Firefox-UI-Fix#455] [Announcements]
## IMPORTANT

Some API has been breaking change.

- `userChrome.tab.on_bottom` -> `userChrome.tabbar.on_bottom`
- `userChrome.tab.on_bottom.above_bookmark` -> `userChrome.tabbar.on_bottom.above_bookmark`
- `userChrome.tab.on_bottom.menubar_on_top` -> `userChrome.tabbar.on_bottom.menubar_on_top`
- `userChrome.tab.centered_label` -> `userChrome.centered.tab.label`

## Changed

**New Options**

-  One liner UI
   - `userChrome.tabbar.one_liner`
   - `userChrome.tabbar.one_liner.combine_navbar`
   - `userChrome.tabbar.one_liner.tabbar_first`
   - `userChrome.tabbar.one_liner.responsive`
-  Auto Hide UI
   - `userChrome.autohide.tab`
   - `userChrome.autohide.tab.opacity`
   - `userChrome.autohide.tab.blur`
   - `userChrome.autohide.tabbar`
   - `userChrome.autohide.navbar`
   - `userChrome.autohide.bookmarkbar`
   - `userChrome.autohide.sidebar`
   - `userChrome.autohide.fill_urlbar`
   - `userChrome.autohide.back_button`
   - `userChrome.autohide.forward_button`
   - `userChrome.autohide.page_action`
   - `userChrome.autohide.toolbar_overlap`
- Hidden UI
   - `userChrome.hidden.tab_icon`
   - `userChrome.hidden.tab_icon.always`
   - `userChrome.hidden.tabbar`
   - `userChrome.hidden.navbar`
   - `userChrome.hidden.sidebar_header`
   - `userChrome.hidden.sidebar_header.vertical_tab_only`
   - `userChrome.hidden.urlbar_iconbox`
   - `userChrome.hidden.bookmarkbar_icon`
   - `userChrome.hidden.bookmarkbar_label`
   - `userChrome.hidden.disabled_menu`
   - `userChrome.tabbar.on_bottom.hidden_single_tab`
- Monospace
   - `userChrome.theme.monospace`
   - `userContent.page.monospace`
-  Centered UI
   - `userChrome.centered.tab`
   - `userChrome.centered.tab.label`
   - `userChrome.centered.urlbar`
   - `userChrome.centered.bookmarkbar`
- Vertical Tab support
   - `userChrome.tabbar.as_titlebar`
   - `userChrome.sidebar.overlap`
   - Icons for [Tree Style Tab](https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab/), [Tab Center Reborn](https://addons.mozilla.org/en-US/firefox/addon/tabcenter-reborn/), [Sidebar Tabs](https://addons.mozilla.org/en-US/firefox/addon/sidebartabs/)

**Bug Fixes**

- URL View: https://github.com/black7375/Firefox-UI-Fix/commit/f6cd9de1807716be5d17632735432717d6d8404b
- Menu rounding: https://github.com/black7375/Firefox-UI-Fix/commit/e2d71ebc18077783864de4b964f838386f98c55e

**Documents**

Various documents have been added, and there is a space where you can share your settings !!

- [Wiki:Tutorial](https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial): Guide for starting `Lepton` custom
- [Wiki:Compatibility Issues Solution](https://github.com/black7375/Firefox-UI-Fix/wiki/Compatibility-Issues-Solution) & [Wiki:Options](https://github.com/black7375/Firefox-UI-Fix/wiki/Options): Add all custom options and screenshot
- [Wiki:Show Off Your Config](https://github.com/black7375/Firefox-UI-Fix/wiki/Show-Off-Your-Config): Lepton custom settings shared by users

---

Discuss the next major version of plan. #453

<hr /><em>This discussion was created from the release <a href='https://github.com/black7375/Firefox-UI-Fix/releases/tag/v6.0.0'>v6.0.0</a>.</em>

**Comment by nikkehtine** [comment]
This is huge, thank you for creating and maintaining this project

**Reply by black7375** [reply]
Yeah, it was a really big change.
We can configure most of the common cases of UI that people want!!
